etcdserver/raft.go

```diff
@@ -188,6 +188,8 @@ func (r *raftNode) start(rh *raftReadyHandler) {
 					return
 				}
 
+				updateCommittedIndex(&ap, rh)
```

Comment:
move this before line 184? or the etcd routine might apply entries before we update commit index. then applied index might be greater than commit index.